code/modules/projectiles/ammo_datums.dm

```diff
@@ -1084,7 +1084,7 @@ datum/ammo/bullet/revolver/tp44
 	damage_falloff = 0.25
 
 /datum/ammo/bullet/sniper/pfc/flak/on_hit_mob(mob/M, obj/projectile/P)
-	staggerstun(M, P, knockback = 4, slowdown = 1.5, stagger = 1, max_range = 17)
+	staggerstun(M, P, knockback = 4, slowdown = 1.5, stagger = 0, max_range = 17)
```

Comment:
This should stay...
It'd be absolutely worthless on large xenos, the damage is piss poor + it takes time to travel on account of being a sniper. You can't reasonably counter that but let's also just face the fact that it's only situationally useful on top of being req-locked (my bad).

The magazine is limited, the firing rate slow enough to the point where you're pretty much bound to miss your target by the time you're ready to fire again, and unlike the Martini (real talk, I strongly disagree with that having it's stagger removed as well; you only get 1 round in the chamber + a lengthy reload) actually requires you to line up your shots or be extraordinarily lucky.

Stagger having a lack of feedback should be dealt with by adding an alert icon to your HUD or by giving a balloon alert when its proc'd.